internal/common/mqtt/mqttconfig.go

```diff
@@ -48,6 +49,7 @@ type Message struct {
 
 var (
 	mqttClient *Client
```

Comment:
I don't think global variable for a Client struct is a good way.

Reply:
similar to above comment,

Reply:
The global variable is removed. All the client data is obtained from hashtable or nil value is returned